packages/@okta/vuepress-site/docs/reference/api/oie-policy-types/index.md

```diff
@@ -2056,3 +2056,141 @@ A Mapping object links together an Application and a Policy.
+          "primary": {
+            "methods": [
+              "EMAIL",
+              "SMS",
+              "VOICE",
+              "PUSH"
+            ]
+          },
+          "stepUp": {
+            "required": false
+          }
+        }
+      },
+      "selfServiceUnlock": {
+        "access": "ALLOW"
+      }
+}
+```
+
+Admin requires end users to verify 2 authenticators before they can recover password. Email and Okta Verify Push can be used by end users to initiate recovery. Security Question is required as step-up.
+
+```json
+"actions": {
+      "passwordChange": {
+        "access": "ALLOW"
+      },
+      "selfServicePasswordReset": {
+        "access": "ALLOW",
+        "requirement": {
+          "primary": {
+            "methods": [
+              "EMAIL",
+              "PUSH"
+            ]
+          },
+          "stepUp": {
+            "required": true,
+            "methods": [
+              "SECURITY_QUESTION"
+            ]
+          }
+        }
+      },
+      "selfServiceUnlock": {
+        "access": "ALLOW"
+      }
+}
+```
+
+Admin requires end users to verify 2 authenticators before they can recover password. Okta Verify Push can be used by end users to initiate recovery. Any authenticator allowed for MFA/SSO is required as step-up.
+
+```json
+"actions": {
+      "passwordChange": {
+        "access": "ALLOW"
+      },
+      "selfServicePasswordReset": {
+        "access": "ALLOW",
+        "requirement": {
+          "primary": {
+            "methods": [
+              "PUSH"
+            ]
+          },
+          "stepUp": {
+            "required": true
+          }
+        }
+      },
+      "selfServiceUnlock": {
+        "access": "ALLOW"
+      }
+}
+```
+
+### Self Service Password Reset Action object
+
+| Property                      | Type    | Description                                                                                 | Supported Values
+| ---                           | ---     | ---                                                                                         | ---
+| `access`                      | String  | Indicates if the action is permitted                                                         | `ALLOW`, `DENY`
+| `requirement`                 | Object  | JSON object containing authenticator methods required to be verified if `access` is `ALLOW` | N/A
+| `requirement.primary.methods` | Array   | Authenticator methods that can be used by end user to initiate password recovery             | `EMAIL`, `SMS`, `VOICE`, `PUSH`
+| `requirement.stepUp.required` | Boolean | If any stepUp verification is required to recover password following primary methods verification | `true`, `false`
+| `requirement.stepUp.methods`  | Array   | If `requirement.stepUp.required` is true, JSON object containing authenticator methods required to be verified as a stepUp. If not specified, any MFA allowed authenticator methods can be used as stepUp. | `SECURITY_QUESTION`
```

Comment:
```suggestion
| `requirement.stepUp.methods`  | Array   | If `requirement.stepUp.required` is true, JSON object containing authenticator methods required to be verified as a step up. If not specified, any MFA allowed authenticator methods can be used as step up. | `SECURITY_QUESTION`
```

> any MFA allowed authenticator methods

Not sure what this means.

Reply:
Similar to above, something like `any enrolled authenticator allowed for sign-on` might be more clear.
Also, do we hyphenate "step-up" or not?  We should pick a style and stick to it.